etc/adminhtml/system.xml

```diff
@@ -363,6 +363,180 @@
+                <group id="omise_offsite_gcash" translate="label" sortOrder="460" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable GCash (Alipay+ Partner) Payment</label>
+                    <comment>Enable customers to pay using GCash payment method</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="461" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
+                        <comment>Enable GCash payment method.</comment>
+                        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                        <config_path>payment/omise_offsite_gcash/active</config_path>
+                    </field>
+                    <field id="title" translate="label" type="text" sortOrder="462" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Title</label>
+                        <comment>This controls the title which the user sees during checkout.</comment>
+                        <config_path>payment/omise_offsite_gcash/title</config_path>
+                    </field>
+                    <field id="allowspecific" translate="label comment" type="select" sortOrder="463" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Allowed Countries</label>
+                        <config_path>payment/omise_offsite_gcash/allowspecific</config_path>
+                        <source_model>Magento\Payment\Model\Config\Source\Allspecificcountries</source_model>
+                        <comment>If not set to all, guest customers will not have a billing country and may not be able to checkout.</comment>
+                    </field>
+                    <field id="specificcountry" translate="label" type="multiselect" sortOrder="464" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Payment from Specific countries</label>
+                        <config_path>payment/omise_offsite_gcash/specificcountry</config_path>
+                        <source_model>Magento\Directory\Model\Config\Source\Country</source_model>
+                        <depends>
+                            <field id="allowspecific">1</field>
+                        </depends>
+                    </field>
+                </group>
+                <group id="omise_offsite_kakaopay" translate="label" sortOrder="470" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Kakaopay (Alipay+ Partner) Payment</label>
+                    <comment>Enable customers to pay using Kakaopay payment method</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="471" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
+                        <comment>Enable Kakaopay payment method.</comment>
```

Comment:
a little here, do we want to keep it consistent and use Kakao Pay here also?

Reply:
good catch! fixed it here https://github.com/omise/omise-magento/pull/309/commits/2a91f6dcb1a397fc8506abc4a9ac7897ab8dcebe